src/insights_client/tests/test_metrics.py

```diff
@@ -120,6 +117,14 @@ def test_proxy_settings_without_auth(rhsm_config_file_factory):
     assert proxy_settings == {"https": "http://localhost:3128"}
 
 
+def test_proxy_settings_with_empty_auth(rhsm_config_file_factory):
```

Comment:
I think we needs tests for a whitespace values too that would verify the _strip_ calls.